README.md

```diff
@@ -7,6 +7,13 @@ Cluster Group Upgrades operator uses the following CRDs:
 
 * **ClusterGroupUpgrade**
 
+and it contains the following controllers:
+
+* **clustergroupupgrade** that's doing both pre-caching and cluster upgrade
```

Comment:
```suggestion
* **clustergroupupgrade** that's doing both preparation steps like pre-caching / backup and the actual cluster upgrade
```

Reply:
I will skip "backup" as I want to have this cherry-picked to the release-4.10 branch.

Reply:
Sure, I didn't know backup wasn't in 4.10